blueocean-dashboard/src/main/js/components/RunDetailsPipeline.jsx

```diff
@@ -151,14 +152,14 @@ export class RunDetailsPipeline extends Component {
 
     // need to register handler to step out of karaoke mode
     // we bail out on scroll up
-    onScrollHandler(elem) {
+    _onScrollHandler(elem) {
```

Comment:
because _handleKeys has it?

Was just trying to make it consistent. I thought it was mainly used for private methods.